app/konnect/dev-portal/applications/dynamic-client-registration/curity.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Configuring Curity for Dynamic Client Registration
+breadcrumb: Curity
+content_type: how-to
+---
+
+
+## Prerequisites
+
+* Enterprise {{site.konnect_short_name}} account.
+* A [Curity account](https://developer.curity.io/)
+* A [Curity instance](https://curity.io/resources/getting-started/) that can be publicly accessed over the internet, or from within the network your Kong Gateways are installed in.
+
+## Configuring Curity
+
+To use dynamic client registration (DCR) with Curity as the identity provider (IdP), there are three important configurations to prepare in Curity. In the following sections you will see how to configure the token issuer, create a client, and enable dynamic client registration for the client.
+
+To get started configuring Curity, log in to your Curity dashboard and complete the following:
+
+1. Select the **Profiles** tab near the top of the dashboard.
+
+2. Select an existing **Token Service Profile** in the **Profiles** diagram, or create a new one if necessary.
+
+3. Complete the following sections using the **Token Service Profile** you have selected.
+
+### Configure token issuer
+
+1. Select **Token Service > Token Issuers** from the menu.
+
+2. Enable the setting **Use Access Token as JWT**.
+
+3. Add a new token issuer using **New Token Issuer** button.
+
+4. Fill in the following values for the token issuer, and click **create**:
+    * Name: `userinfo`
+    * Issuer Type: `jwt`
+    * Purpose Type: `userinfo`
+
+5. In the displayed "Edit Custom Token Issuer" form, select the desired values for **Tokens Data Source ID**, **Signing Key**, and **Verification KeyStore**.
+
+### Create a client
+
+1. Select **Token Service > Clients** from the menu.
+
+2. Click **New Client** to add a new client.
+
+3. Give the client a unique and descriptive name, noting it for later use.
+
+4. Click the **Capabilities** in the overview diagram to add a capability to the **client**.
+
+5. Select **Client Credentials** and click **next**.
+
+6. Set the **Authentication Method** to `secret` and generate a secret, copy it for later use, and click **next**.
+
+   {:.important}
+   > **Important:** Store the secret in a place you can reference, because it will not be visible after this step.
+
+### Enable Dynamic Client Registration
+
+1. Select **Token Service > General > Dynamic Registration** from the menu.
+
+2. Click **Enable Dynamic Client Registration**.
+
+3. Ensure **Non Templatized** and **Dynamic Client Management** are both enabled and click **next**.
```

Comment:
The `kong.Spelling` check is flagging this, but this is how it appears in the Curity UI
<img width="877" alt="Screen Shot 2022-12-12 at 2 53 48 PM" src="https://user-images.githubusercontent.com/8764246/207141266-49ccc54c-56a5-44ac-b7aa-98055f3e5127.png">

Reply:
I can fix that, I'll just add it to the Vale dictionary.